tests/test_tux.py

```diff
@@ -0,0 +1,102 @@
+import os
+import logging
+import tempfile
+import unittest
+
+from squad_client.tux import build_test_name, buildset_test_name, load_builds, combined_kconfig_sha
+
+
+class TuxTest(unittest.TestCase):
+
+    def setUp(self):
+        self.root_dir = os.path.join("tests", "data", "tux")
+        self.assertTrue(os.path.exists(self.root_dir))
+
+        self.build_dir = os.path.join(self.root_dir, "build-x86-gcc")
+        self.assertTrue(os.path.exists(self.build_dir))
+
+        self.buildset_dir = os.path.join(self.root_dir, "buildset-x86")
+        self.assertTrue(os.path.exists(self.buildset_dir))
+
+        self.tmp_dir = tempfile.mkdtemp()
+        self.assertTrue(os.path.exists(self.tmp_dir))
+
+    def test_load_builds_with_build(self):
+        builds = load_builds(
+            os.path.join(self.build_dir, "build.json")
+        )
+
+        self.assertTrue(len(builds) == 1)
```

Comment:
It would be a nice idea if you could cross check ALL expected fields.

Reply:
@chaws It's hard to keep up with the tux team on what fields are exactly in the build.json, but perhaps we should keep a list of required fields (ie, fields in each build we depend on) and phase out the json schema validation?

Reply:
Same reason as https://github.com/Linaro/squad-client/pull/137